core-it-support/core-it-javaagent/pom.xml

```diff
@@ -53,6 +53,17 @@ under the License.
             <goals>
               <goal>shade</goal>
             </goals>
+            <configuration>
+              <filters>
+                <filter>
+                  <artifact>*:*</artifact>
+                  <excludes>
+                    <exclude>module-info.class</exclude>
+                    <exclude>META-INF/MANIFEST.MF</exclude>
```

Comment:
m-jar-p adds items to MANIFEST - see line 40-43 - we need its, so we don't drop MANIFEST
we can try https://maven.apache.org/plugins/maven-shade-plugin/examples/resource-transformers.html#ManifestResourceTransformer

Reply:
Nice catch, thanks! I would have hoped that "something" would fail the build if those manifest entries are missing -- seeing the linked PR, I assume this is only failing with Maven 4.x, which is currently not part of PR checks.

In any case, I adjusted the filter to exclude only the `asm` artifacts, which should fix the issue (the build still won't pass with Maven 4.x, but the reported errors/failures seem to be consistent with the ones I am seeing on master branch)